app/build.gradle

```diff
@@ -3,10 +3,10 @@ apply plugin: 'com.android.application'
 android {
     signingConfigs {
         config {
-            keyAlias 'MangoReaderKey'
-            keyPassword 'uC7t2zNQySAC'
-            storeFile file('C:/Users/Clarence/Documents/MangoReader.alpha.release.jks')
-            storePassword 'uC7t2zNQySAC'
+//            keyAlias 'MangoReaderKey'
```

Comment:
Why did we comment this out again? Actually why did we include it in the first place?